modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/frontend/data/set/filter/CommerceAccountAutocompleteFDSFilter.java

```diff
@@ -28,13 +29,23 @@
 	service = FDSFilter.class
 )
 public class CommerceAccountAutocompleteFDSFilter
-	extends BaseAutocompleteFDSFilter {
+	extends BaseSelectionFDSFilter {
```

Comment:
All classes that extend `BaseSelectionFDSFilter` should probably be `*SelectionFDSFilter`, I doubt this would pass Brian :slightly_smiling_face: . This one should be `CommerceAccountSelectionFDSFilter`.

Reply:
Sorry I missed this, makes sense. On it!

Reply:
Since we're splitting the task into different parts, I've left the migration in another branch and added these changes there to make it easier once I've left. See here https://github.com/tinycarol/liferay-portal/compare/LPS-170299...tinycarol:liferay-portal:LPS-170299-migrations